### ISSUE TITLE
Google Verbatim functionality

### ISSUE THREAD

**Issue by grahamperrin** [thombles/MoreVerbatim#1]
Hi

From <https://addons.mozilla.org/addon/moreverbatim/>: 

> … There's a Verbatim mode that used to help but now it simply doesn't work. …

It nearly always _does_ work (does significantly refine results), for me. 

Maybe reword the description for the extension. Thanks.

**Comment by thombles** [comment]
Aha, that's good that they fixed it. It was very broken at the time I wrote this. I haven't been using either Firefox or Google for a little while so this extension is effectively unmaintained. I've disabled the addon and I'll archive the repo in case anyone needs this workaround in future. Thanks for the update!
